demos/smart_classroom_demo_gapi/main.cpp

```diff
@@ -970,106 +1441,95 @@ int main(int argc, char* argv[]) {
-                                      actions_map, face_id_to_label_map,
-                                      face_obj_id_to_smoothed_action_maps);
-            }
+        if (state::work_num_frames > 0) {
+            slog::info << "Mean FPS: " << static_cast<float>(state::work_num_frames) /
+                                          static_cast<float>(avg.elapsed().ss) << slog::endl;
         }
+        slog::info << "Frames processed: " << state::total_num_frames << slog::endl;
+
+        // if (actions_type == STUDENT) {
+        //     auto face_tracks = config::tracker_reid.vector_tracks();
+        // 
+        //     // correct labels for track
+        //     std::vector<Track> new_face_tracks = UpdateTrackLabelsToBestAndFilterOutUnknowns(face_tracks);
+        //     std::map<int, int> face_track_id_to_label = GetMapFaceTrackIdToLabel(new_face_tracks);
+        // 
+        //     std::vector<std::string> face_id_to_label_map = {}; // face_recognizer->GetIDToLabelMap();
+        // 
+        //     if (!face_id_to_label_map.empty()) {
+        //         std::map<int, FrameEventsTrack> face_obj_id_to_actions_track;
+        //         ConvertActionMapsToFrameEventTracks(face_obj_id_to_action_maps, default_action_index,
+        //                                             &face_obj_id_to_actions_track);
+        // 
+        //         // const int start_frame = 0;
+        //         const int end_frame = face_obj_id_to_action_maps.size();
+        //         std::map<int, RangeEventsTrack> face_obj_id_to_events;
+        //         // SmoothTracks(face_obj_id_to_actions_track, start_frame, end_frame,
+        //         //              smooth_window_size, smooth_min_length, default_action_index,
+        //         //              &face_obj_id_to_events);
+        // 
+        //         slog::info << "Final ID->events mapping" << slog::endl;
+        //         logger.DumpTracks(face_obj_id_to_events,
+        //                           actions_map, face_track_id_to_label,
+        //                           face_id_to_label_map);
+        // 
+        //         std::vector<std::map<int, int>> face_obj_id_to_smoothed_action_maps;
+        //         ConvertRangeEventsTracksToActionMaps(end_frame, face_obj_id_to_events,
+        //                                              &face_obj_id_to_smoothed_action_maps);
+        // 
+        //         slog::info << "Final per-frame ID->action mapping" << slog::endl;
+        //         logger.DumpDetections(video_path, frame.size(), work_num_frames,
+        //                               new_face_tracks,
+        //                               face_track_id_to_label,
+        //                               actions_map, face_id_to_label_map,
+        //                               face_obj_id_to_smoothed_action_maps);
+        //     }
+        // }
 
         std::cout << presenter.reportMeans() << '\n';
     }
```

Comment:
This `main` is way to huge. Need to think how to restructure it.

Reply:
Kernels and Ops left from main.